src/sentry/grouping/api.py

```diff
@@ -121,14 +121,14 @@ def _get_enhancements(self, project) -> str:
         cache.set(cache_key, rv)
         return rv
 
-    def _get_config_id(self, project):
+    def _get_config_id(self, project: Project):
         raise NotImplementedError
 
 
 class ProjectGroupingConfigLoader(GroupingConfigLoader):
     option_name: str  # Set in subclasses
 
-    def _get_config_id(self, project):
+    def _get_config_id(self, project: Project):
```

Comment:
You need a return type. Basically every time there's no `) -> :`

Reply:
Yeah - most of these were cases where the typing was more complex than a 30s change, this at least gets us _closer_